packages/page-objects/src/components/editor/TextEditor.ts

```diff
@@ -626,6 +626,32 @@ export class TextEditor extends Editor {
 		return new Breakpoint(breakpoints[0], lineElement);
 	}
 
+	/**
+	 * Get all breakpoints.
+	 * @returns List of Breakpoints.
+	 */
+	async getBreakpoints(): Promise<Breakpoint[]> {
+		const breakpoints: Breakpoint[] = [];
+
+		const breakpointLocators = Breakpoint.locators.TextEditor.breakpoint;
+		const breakpointContainer = TextEditor.versionInfo.version >= '1.80.0' ? await this.findElement(By.className('glyph-margin-widgets')) : this;
```

Comment:
this would be better to move into locators.. there is always a diff file  indicating version and all upcoming versions where the changes are necessary.. see https://github.com/redhat-developer/vscode-extension-tester/tree/main/packages/locators/lib

in this case, I would expect new 1.80.0.ts file with appropriate changes

Reply:
There's lot more of similar cases so I opened issue for this to fix it together. 

https://github.com/redhat-developer/vscode-extension-tester/issues/1504